numba/np/npdatetime.py

```diff
@@ -847,3 +847,7 @@ def impl(x):
                 return np.int64(-2)
             return np.int64(x)
     return impl
+
+
+lower_builtin(npdatetime_helpers.datetime_minimum, types.VarArg(types.Any))(datetime_minimum_impl)
+lower_builtin(npdatetime_helpers.datetime_maximum, types.VarArg(types.Any))(datetime_maximum_impl)
```

Comment:
I would be more specific on the input types to avoid future issues.

```suggestion
lower_builtin(npdatetime_helpers.datetime_minimum, types.NPDatetime, types.NPDatetime)(datetime_minimum_impl)
lower_builtin(npdatetime_helpers.datetime_minimum, types.NPTimedelta, types.NPTimedelta)(datetime_minimum_impl)
lower_builtin(npdatetime_helpers.datetime_minimum, types.NPDatetime, types.NPDatetime)(datetime_maximum_impl)
lower_builtin(npdatetime_helpers.datetime_minimum, types.NPTimedelta, types.NPTimedelta)(datetime_maximum_impl)
```